src/v2/DevTools/mockStripe.ts

```diff
@@ -0,0 +1,31 @@
+const mockElement = () => ({
+  mount: jest.fn(),
+  destroy: jest.fn(),
+  on: jest.fn(),
+  update: jest.fn(),
+})
+
+const mockElements = () => {
+  const elements = {}
+  return {
+    create: jest.fn(type => {
+      elements[type] = mockElement()
+      return elements[type]
+    }),
+    getElement: jest.fn(type => {
+      return elements[type] || null
+    }),
+  }
+}
+
+export const mockStripe = () => ({
+  elements: jest.fn(() => mockElements()),
+  createToken: jest.fn(),
+  createSource: jest.fn(),
+  createPaymentMethod: jest.fn(),
+  confirmCardPayment: jest.fn(),
+  confirmCardSetup: jest.fn(),
+  paymentRequest: jest.fn(),
+  handleCardAction: jest.fn(),
+  _registerWrapper: jest.fn(),
+})
```

Comment:
Mentioned elsewhere, but this util could possibly exist in concert with the lib being mocked in `__mocks__`